### PR TITLE
added name to readme

### PR DESCRIPTION
**Pull request by sbue**

### CODE REVIEW

**Comment by abhisuri97** [comment]
No caps

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # cls
 Community Legal Services Project [Fall-2016]
+SANTIAGO BUENAHORA
```
Comment (abhisuri97) [inline README.md]: Please No caps (also testing out this cool review feature amaze).

Comment (sbue) [reply README.md]: So should I pull again (w/ no caps)?

Comment (abhisuri97) [reply README.md]: You can just commit now and it will appear with changes here.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # cls
 Community Legal Services Project [Fall-2016]
-SANTIAGO BUENAHORA
+Santiago Buenahora
```
Comment (abhisuri97) [inline README.md:3]: LGTM 👍

**Pull request merged**